Sources/ComposableArchitecture/Documentation.docc/Articles/Performance.md

```diff
@@ -74,10 +74,92 @@ This means `AppView` does not actually need to observe any state changes. This v
 created a single time, whereas if we observed the store then it would re-compute every time a single
 thing changed in either the activity, search or profile child features.
 
-If sometime in the future we do actually need some state from the store, we can create a localized
-"view state" struct that holds only the bare essentials of state that the view needs to do its
-job. For example, suppose the activity state holds an integer that represents the number of 
-unread activities. Then we could observe changes to only that piece of state like so:
+If sometime in the future we do actually need some state from the store, we can start to observe
```

Comment:
this is an update to the performance article. I realized that first talked about a `selectedTab` piece of state and then later an `unreadActivityCount` state was more natural than the reverse.